### ISSUE TITLE
Markdown files using guide layout does not support ordered and unordered entries in the same list

### ISSUE THREAD

**Issue by anthony-chu** [electricjs/electric#36]
When adding a markdown page using the guide layout, lists that have both ordered and unordered list items don't render properly.

```markdown
1. Numbered item 1
    - child item 1
1. Numbered item 2
```

should render as

1. Numbered item 1
    - child item 1
1. Numbered item 2

but instead renders as

```
1. Numbered item 1
    - child item 1
3. Numbered item 2
```

the next numbered list item should be 2, but instead a 3 is displayed, as it appears to be counting the "-" as another ordered item (even though there's an indentation that denotes otherwise).

**Comment by zenorocha** [comment]
Sounds like a problem related to Soy rendering, can you check @Robert-Frampton?

**Comment by Robert-Frampton** [comment]
Hey @anthony-chu 

Can you provide entire markdown file for context?

**Comment by anthony-chu** [comment]
hey @Robert-Frampton, I made it a gist [here](https://gist.github.com/anthony-chu/05394140dc224013ac6e52a2f98c331d).

**Comment by Robert-Frampton** [comment]
Looks like the nested list elements were just indented too much (each had 8 spaces). If you indent the nested lists with a tab, or with 4 spaces it'll work.

<img width="431" alt="screen shot 2017-03-13 at 3 41 44 pm" src="https://cloud.githubusercontent.com/assets/2466445/23878266/9eadeb7e-0803-11e7-9270-a5a80f618428.png">

**Comment by Robert-Frampton** [comment]
Hey @anthony-chu @zenorocha 

I found the root cause of the issue, the generated markup is actually fine, Marble is actually what's causing the issue with the ordered lists not displaying the correct number.

Normally the browser displays the number, but for whatever reason Marble hides it and does some custom styling and uses the `:before` content of of the [list item](https://github.com/wedeploy/marble/blob/master/src/templates/docs/_guide.scss#L134) to display it.

**Comment by zenorocha** [comment]
Nice! @ygorcosta can you take a look?

**Comment by ygorcosta** [comment]
hey @zenorocha, I've fixed that on [marble v2.9.6](https://github.com/wedeploy/marble/releases/tag/v2.9.6)

@anthony-chu can you update your marble version and test?
